apps/shared/interval.h

```diff
@@ -18,6 +18,8 @@ class Interval {
   void setEnd(double f);
   void setStep(double f);
   void setElement(int i, double f);
+  bool needCompute() const { return m_needCompute; }
```

Comment:
This does't need to be exposed, does it?

Reply:
We don't need it, you're right.
I always add setter and getter, even when they're useless